pywr/core.py

```diff
@@ -118,9 +132,22 @@ def __init__(self, solver=None, parameters=None):
 
         self.node = {}
         self.group = {}
+        self.scenarios = ScenarioCollection()
 
         self.reset()
 
+    @property
+    def number_of_timesteps(self):
+        return len(self.timestepper)
+
+    @property
+    def number_of_scenarios(self):
```

Comment:
This doesn't feel very Pythonic. I wonder if we could do something more like `len(model.scenarios)` and `len(model.scenarios.combinations)`? Not sure.